debian/install

```diff
@@ -21,6 +21,8 @@ config/gtk-2.0 etc/skel/.config/
 config/dconf etc/skel/.config/
 config/console usr/share/kano-desktop/config
 
+config/sudoers.d /etc
```

Comment:
I had the same thought @tombettany but then I saw this:

 * https://github.com/KanoComputing/kano-desktop/blob/3907521c460fc6a9b6ce50ff6f0a836981d9692a/debian/install#L3